proto/tendermint/p2p/pex.proto

```diff
@@ -17,21 +15,9 @@ message PexResponse {
   repeated PexAddress addresses = 1 [(gogoproto.nullable) = false];
 }
 
-message PexAddressV2 {
-  string url = 1 [(gogoproto.customname) = "URL"];
-}
-
-message PexRequestV2 {}
-
-message PexResponseV2 {
-  repeated PexAddressV2 addresses = 1 [(gogoproto.nullable) = false];
-}
-
 message PexMessage {
   oneof sum {
     PexRequest    pex_request     = 1;
     PexResponse   pex_response    = 2;
```

Comment:
what are your thoughts on versioning the proto files? seems like this could be a good first use case for this?

Reply:
> what are your thoughts on versioning the proto files? seems like this could be a good first use case for this?

I'm ambivalent about versioning the file. I don't really see what problems adding another version marker would solve. 

If we break the wire format, we'll have to update the software version anyway, and if we don't break the wire format, then the existing software should continue to work without changes. That's the original reason for the schema language having explicit field tags. 

Of course, we might add new fields and need to update the software to take advantage of them, but at that point the version the client cares about is the version of the service it's talking to, not the schema file. 

IMO wire-format changes are the only ones that matter. Even if the generated code changes, old versions of the library should continue to work as well as they ever did. If we want to change how the service interprets old messages (e.g., we cut out some feature and start reporting errors) then the client has to update because the _service_ version changed.